src/content/app/tools/blast/views/blast-form/BlastForm.tsx

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
```

Comment:
This might be slightly out of scope, but since it should be a minor change can we switch to `useAppSelector` and `useAppDispatch`?

Reply:
Thats done.